### ISSUE TITLE
Add instructions for per user installation to readme

### ISSUE THREAD

**Issue by 00face** [stnkl/EverythingToolbar#229]
I need portable mode without install, so I can use this on my work computer.

**Comment by stnkl** [comment]
Version [0.6.2](https://github.com/stnkl/EverythingToolbar/releases/tag/0.6.2) actually supports installing per user without admin rights but it was changed to per machine due to #120.

I could probably supply a script that allows installing per user and add instructions to the readme. The installer however should always work reliably for everybody so I don't see it returning as an installer option at the moment.

**Comment by jimbrig** [comment]
I'm confused as to how i can install without admin privileges. V0.6.2 msi installer doesn't work and neither does the `regasm` approach with the toolbar `.dll`. How can I install without Admin?

**Comment by Horst12** [comment]
A Toolbar install without admin rights seems useless for me
because a Everything installion without admin rights can only use folder indexing.
So most benefits of Everything are unavailable.

**Comment by jimbrig** [comment]
ive installed everything via scoop without admin rights and it works the same as my normal machine wide installation on my personal laptop (there's also everything.portable)

**Comment by Horst12** [comment]
Even Everything portable needs admin rights if you want NTFS indexing.
It just doesn't have the service.

**Comment by stnkl** [comment]
> I'm confused as to how i can install without admin privileges. V0.6.2 msi installer doesn't work and neither does the `regasm` approach with the toolbar `.dll`. How can I install without Admin?

Can you please explain in what way the 0.6.2 installer does not work? I just tried again and I could install it without admin rights. The batch file method proposed by @Horst12 only works with admin privileges. I will try to update the instructions for user-installation this week.

**Comment by stnkl** [comment]
Actually I found a user installation script that I still had. I extracted the latest release and put the `install.cmd` and `uninstall.cmd` in there. To install it extract it to a safe place and run the `install.cmd` and see if it works.

[EverythingToolbar-0.7.4_user_install.zip](https://github.com/stnkl/EverythingToolbar/files/8413431/EverythingToolbar-0.7.4_user_install.zip)

But now that I tried it again on my machine I actually have the same problem that was reported in #120. So I can actually reproduce this now (for whatever reason). It will take me some time to diagnose this though since I'm fairly busy at the moment.
